pynamodb/connection/base.py

```diff
@@ -928,39 +847,13 @@ def update_item(self,
             operation_kwargs.update(self.get_item_collection_map(return_item_collection_metrics))
         if return_values:
             operation_kwargs.update(self.get_return_values_map(return_values))
-        if not actions and not attribute_updates:
-            raise ValueError("{0} cannot be empty".format(ATTR_UPDATES))
+        if not actions:
+            raise ValueError("'actions' cannot be empty")
         actions = actions or []
```

Comment:
technically not from `Model`, but it's still taken as a kwarg and defaults to `None` so it might be better to keep this for now (short of a larger refactor)

Reply:
the unnecessary line is `actions = actions or []`

Reply:
oh duh, missed the exception above. removing.